src/functions/software-registry/delete-software/handler.ts

```diff
@@ -0,0 +1,46 @@
+import { APIGatewayProxyEvent, APIGatewayProxyHandler } from "aws-lambda";
+import { DocumentClient } from "aws-sdk/clients/dynamodb";
+import SoftwareService from "src/apis/software-service";
+import { middyfy } from "src/libs/lambda";
+
+const dynamoDb = new DocumentClient();
+const softwareService = new SoftwareService(dynamoDb);
+
+/**
+ * Handler for deleting a software entry in DynamoDB.
+ * 
+ * @returns Response object with status code and body.
+ */
+export const deleteSoftwareHandler: APIGatewayProxyHandler = async (event: APIGatewayProxyEvent) => {
```

Comment:
Use middy

Reply:
i have wrapped the deleteSoftwareHandler with middy in the main export